src/ch5/QDemo.java

```diff
@@ -0,0 +1,50 @@
+package ch5;
+
+public class QDemo {
+    public static void main(String[] args) {
+        Queue bigQ = new Queue(26);
+
+        char sumb;
+        int id;
+
+        System.out.println("Используем очередь для помещения в нее английского алфавита");
+        for (id = 0; id < 26; id++)
+            bigQ.put((char) ('A' + id));
+
+        System.out.println("Содержание очереди:  ");
+        for (id = 0; id < 26; id++) {
+            sumb= bigQ.get();
+            if (sumb != (char) 0) System.out.print(sumb);
+        }
+    }
+}
+class Queue {
+    private char[] q;
```

Comment:
Unintelligible names (for example, consisting of only one letter) are used only in cycles as counters
For everything else, you need to come up with meaningful names.

q -> queueArray/queueArr/longArray

The name should describe the purpose of the variable in short form.